### PR TITLE
Update security-assessment.md

### PR DESCRIPTION
**Pull request by szymonkowalczykzf**
Updated the assessment Report according to our last meeting.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: docs/security-assessment.md [hunk 1]
```diff
@@ -46,38 +46,49 @@ flowchart TD
...
-    D7(Product Measures Visualization \n Just visualization of the data \n Out of scope - Not yet developped)
-
+ 
     D(Stock View Controller)
     D2(EDC View Controller)
-    D3(Data Request Controller)
+    D3(Data Request Controller - Separtate \n Instance for each controller like \n Stock / Demand / Production / Delivery)
     D4(Data Response Controller)
     D5[(PostgreSQL DB \n Main Database)]
+    D8(Data Twin Registry)
```
Comment (tom-rm-meyer-ISST) [inline docs/security-assessment.md:62]: "Digital Twin Registry (DTR)" instead of "Data Twin Registry"

**Review by tom-rm-meyer-ISST — CHANGES_REQUESTED** [review]
Thanks for the PR, Found one typo. Please correct.

File: docs/security-assessment.md (hunk 1 above)
Comment (szymonkowalczykzf) [reply docs/security-assessment.md]: Fixed, thanks for the review.

**Review by tom-rm-meyer-ISST — APPROVED** [review]
LGTM, thanks for your support!

**Pull request merged**